src/tests/ftest/util/test_utils_container.py

```diff
@@ -498,17 +498,10 @@ def list_snaps(self):
         Returns:
             list: a list of snapshot epochs defined for this container
         """
-        self.log.info("Listing snapshots for container: %s", str(self))
         if not self.daos:
-            raise DaosTestError("Undefined daos command")
-
-        # destroy snapshot using daos utility
-        kwargs = {
-            "pool": self.pool.identifier,
-            "cont": self.identifier,
-        }
-        self._log_method("daos.container_list_snaps", kwargs)
-        data = self.daos.container_list_snaps(**kwargs)
+            raise DaosTestError(
+                "Undefined daos command for listing snapshots for container {}".format(self))
```

Comment:
FWIW, `self.daos` will always be defined once I finish updating all test code to use `get_container`